manifest.lkml

```diff
@@ -3,3 +3,10 @@ project_name: "spoke-default"
 local_dependency: {
   project: "looker-hub"
 }
+
+# LAMS
+# rule: no_view_wildcards {
+#  description: "Please don't use wildcard view include (e.g. include: 'views/*') to avoid increasing LookML verification time. Explicitly list views to be included."
+#  match: "$.file.*..include.*"
+#  expr_rule:  ($not ($boolean ($match "views/.*\\*" ::match))) ;;
```

Comment:
Annoyingly it turns out that in the JSON-ified project data LAMS is matching against, if a file contains a single include then the value of `include` will be a string (not a list), which the JSONPath expression `include.*` won't match.

Here's a modification to the rule to handle `include` potentially being a string or list, which I've successfully tested locally (it finds one such single-wildcard-include case in `duet/mr1DEVELOPMENT/explores.explore.lkml`):
```suggestion
#  match: "$.file.*..include"
#  expr_rule:
#   ($let includes ($if ($is-list match) match ($list match)))
#   ($not ($any ...($map includes (-> (include) ($boolean ($match "views/.*\\*" include))))))
#  ;;
```

Reply:
Or if you go with a 2-space indent like I suggested in another comment:
```suggestion
#   match: "$.file.*..include"
#   expr_rule:
#     ($let includes ($if ($is-list match) match ($list match)))
#     ($not ($any ...($map includes (-> (include) ($boolean ($match "views/.*\\*" include))))))
#   ;;
```